contracts/DNSRegistrar.sol

```diff
@@ -0,0 +1,133 @@
+pragma solidity ^0.4.23;
+
+import "@ensdomains/ens/contracts/ENS.sol";
+import "@ensdomains/dnssec-oracle/contracts/Buffer.sol";
+import "@ensdomains/dnssec-oracle/contracts/DNSSEC.sol";
+import "@ensdomains/dnssec-oracle/contracts/BytesUtils.sol";
+import "@ensdomains/dnssec-oracle/contracts/RRUtils.sol";
```

Comment:
nice when we finally have everything scoped

Reply:
👍